app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    text = command.split(" ")
+    if text[0] != "cp":
+        return
+    if text[1] == text[2]:
+        return
+    with open(text[1], "r") as f1, open(text[2], "w") as f2:
+        f2.write(f1.read())
```

Comment:
Use more convenient naming 
```suggestion
    with open(text[1], "r") as file1, open(text[2], "w") as file2:
        file2.write(file1.read())
```